ral-facilities/datagateway
# Feature/display search numbers #419

## Description
Added badges to each tab of the search view to show the number of search results of that type.

## Testing instructions
Add a set up instructions describing how the reviewer should test the code

- [x] Check the badge number is accurate for all tabs after searching
EDIT:
- [ ] Check alignment for different browser font sizes

## Agile board tracking
closes #419
